builders/build/customize/customizing-chain-specs.md

```diff
@@ -29,16 +29,16 @@ The client specification contains the configuration of the network and other set
 - **Boot Nodes** - set of boot nodes that will be used when the new node joins the network and syncs
 - **Telemetry Endpoints** - an optional list of endpoints to send information and monitor the operation of the network
 - **Protocol Id** - a unique name defining the network protocol
-- **Relay Chain** - defines the relay chain the Tanssi appchain interacts with
-- **Parachain Id** - sets the parachain id that has been reserved and assigned in the relay chain
+- **Relay Chain** - defines the id of the orchestration chain the Tanssi appchain interacts with
```

Comment:
maybe we should ask for clarification around the parachain and relay chain terminology?

Reply:
In this case we are referring to key-value parameters in the specs file, and the keys are "relay_chain" and "para_id" (That's why I only edited the description)